src/NServiceBus.Core/Serialization/SerializationFeature.cs

```diff
@@ -51,8 +51,8 @@ protected internal sealed override void Setup(FeatureConfigurationContext contex
             var resolver = new MessageDeserializerResolver(defaultSerializer, additionalDeserializers);
 
             var logicalMessageFactory = new LogicalMessageFactory(messageMetadataRegistry, mapper);
-            context.Pipeline.Register(new DeserializeLogicalMessagesConnector(resolver, logicalMessageFactory, messageMetadataRegistry, mapper), "Deserializes the physical message body into logical messages");
-            context.Pipeline.Register(new SerializeMessageConnector(defaultSerializer, messageMetadataRegistry), "Converts a logical message into a physical message");
+            context.Pipeline.Register("DeserializeLogicalMessagesConnector", new DeserializeMessageConnector(resolver, logicalMessageFactory, messageMetadataRegistry, mapper), "Deserializes the physical message body into logical messages");
+            context.Pipeline.Register("DeserializeMessageConnector", new SerializeMessageConnector(defaultSerializer, messageMetadataRegistry), "Converts a logical message into a physical message");
```

Comment:
Shouldn't this be "SerializeMessageConnector`
```suggestion
            context.Pipeline.Register("SerializeMessageConnector", new SerializeMessageConnector(defaultSerializer, messageMetadataRegistry), "Converts a logical message into a physical message");
```

Reply:
absolutely, no idea how I messed that up. Good catch 👍